homeassistant/components/notify/group.py

```diff
@@ -0,0 +1,66 @@
+"""
+Group platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.group/
+"""
+import collections
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (CONF_PLATFORM, CONF_NAME,
+                                 CONF_ENTITY_ID)
+from homeassistant.components.notify import (DOMAIN, ATTR_MESSAGE, ATTR_DATA,
+                                             BaseNotificationService)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ENTITIES = "entities"
+
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): "group",
+    vol.Required(CONF_NAME): vol.Coerce(str),
+    vol.Required(CONF_ENTITIES): vol.All(cv.ensure_list, [{
+        vol.Required(CONF_ENTITY_ID): vol.Any(cv.string, None),
```

Comment:
This should not be entity id because it is not one. It's a service name. By naming it entity id we will confuse users.